javalanglib/src/main/scala/java/lang/Character.scala

```diff
@@ -213,6 +213,17 @@ object Character {
     else -1
   }
 
+  // ported from https://github.com/gwtproject/gwt/blob/master/user/super/com/google/gwt/emul/java/lang/Character.java
+  def forDigit(digit: Int, radix: Int): Char = {
+    if ( radix < MIN_RADIX || radix > MAX_RADIX || digit < 0 || digit >= radix) {
+      0
+    }
+    else {
+      val overBaseTen = digit - 10
+      (if (overBaseTen < 0) '0' + digit else 'a' + overBaseTen).toChar
```

Comment:
It would be be possible, yes - but I wonder on the benefit/performance? The check to return 0 has to stay as JS throws rather than return 0. So in effect, this is only one line of code. My hunch would be to keep it as is - but your call.